doc/index.rst

```diff
@@ -35,19 +32,41 @@ interface toolkits.
    </div>
    <span class="clear_screenshots"></span>
 
-Matplotlib tries to make easy things easy and hard things possible.  You
-can generate plots, histograms, power spectra, bar charts, errorcharts,
-scatterplots, etc., with just a few lines of code.  For examples, see the
-:doc:`sample plots <tutorials/introductory/sample_plots>` and :doc:`thumbnail
-gallery <gallery/index>`.
+Matplotlib makes easy things easy and hard things possible.
```

Comment:
I disagree here. The first sentence should tell what the library is about. Looking at other libraries, this seems to be common practice:

- NumPy is the fundamental package for scientific computing with Python.
- pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures and data analysis tools for the Python programming language.
- Seaborn is a Python data visualization library based on matplotlib.
- Bokeh is an interactive visualization library for modern web browsers.
- ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics.

Reply:
In that vein, the phrasing should probably be something like "Matplotlib is a comprehensive library for for building static, animated, and interactive visualizations in Python" or from the grant
" Matplotlib is the fundamental visualization library for the scientific Python Ecosystem" but I'm not too hung up on this

Reply:
I've taken your first proposal, but with "library for creating ..." instead of "building". The latter sounds too technical and like a lot of effort. :smile: